ladybug/datacollection.py

```diff
@@ -3,7 +3,13 @@
 from .datatype import DataPoint
 
 from collections import OrderedDict
-from itertools import izip
+
+from builtins import range
```

Comment:
@ocni-dtu this will fail in Python 2 as there is no `builtins` module.

Reply:
Correct Py2 doesn't ship with builtins it is a part of the future package. See this: http://python-future.org/compatible_idioms.html#setup

Reply:
our challenge is to keep the code compatible with IronPython too. Otherwise we can't use the libraries directly from Grasshopper interface.